apps/fxc-tiles/assets/download.sh

```
@@ -1,10 +1,12 @@
 #! /usr/bin/env sh
 
-DIST_ASP_FOLDER=`realpath $(dirname "$0")`
-DST_FOLDER=`realpath $DIST_ASP_FOLDER/../../../../apps/fxc-tiles/src/assets/airspaces`
+APP_FOLDER=`realpath $(dirname "$0")/..`
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)